src/site/pages/accessibility-guide.md

```diff
@@ -37,49 +37,62 @@ Users with poor motor control can use a range of devices to access contents. Use
 
 Users who have difficulty processing information benefit from well-written content. Information should clear, concise, and easy to scan. Consider visual hierarchy, chunk content into short, related sections, and avoid long paragraphs.
 
+## Level of Support
```

Comment:
I like the way you broke this stuff out. I'm hesitating on this header title though. What about "Designing and Developing for Accessibility"? More general and broadly introduces this section.